patches/components-os_crypt-os_crypt_win.cc.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/components/os_crypt/os_crypt_win.cc b/components/os_crypt/os_crypt_win.cc
+index 792233d4405f..e9fee5024fcb 100644
+--- a/components/os_crypt/os_crypt_win.cc
++++ b/components/os_crypt/os_crypt_win.cc
+@@ -26,6 +26,11 @@ bool OSCrypt::DecryptString16(const std::string& ciphertext,
+ 
+ bool OSCrypt::EncryptString(const std::string& plaintext,
+                             std::string* ciphertext) {
++  if (IsEncryptionDisabled()) {
```

Comment:
this can be further reduced by passing the pointer to ciphertext to `IsEncryptionDisabled` - the smaller we can make the patches the better so `if (IsEncryptionDisabled(ciphertext)) { return true; }` would be ideal

Reply:
Done